### ISSUE TITLE
Various color issues in dark mode after Qt 6.4.0 update (Manjaro Linux)

### ISSUE THREAD

**Issue by kleinerpirat** [ankitects/anki#2133]
Related to #2124.
The screenshots are from a build **without custom Qt stylesheets.**

---

- [x] progress bar is black:

![image](https://user-images.githubusercontent.com/62722460/195803873-917e5243-5c31-4718-9a20-c5a809b5c4f0.png)

- [x] highlight background for items of QListView, QTableView, QTreeView, QComboBox, QMenu

![image](https://user-images.githubusercontent.com/62722460/195804207-c45d0d2c-08cc-4fbb-8333-dc2e3a5ffd0f.png)
![image](https://user-images.githubusercontent.com/62722460/195806039-69055336-de8b-42f9-8efe-47043c9e35d8.png)
![image](https://user-images.githubusercontent.com/62722460/195804823-b40c8fbe-14e3-4fce-bb23-96b868e20f05.png)

The [ColorRole](https://doc.qt.io/qt-6/qpalette.html#ColorRole-enum) enum of QPalette doesn't contain entries for these elements.

Again, setting these explicitly via stylesheet would fix this, but it does come with caveats. For example, it's not possible to animate things in Qt stylesheets, so native animations for QProgressBar would be lost.

**Comment by dae** [comment]
Have only had a chance to test on a Mac so far, but if I comment out this line:

```
        palette.setColor(QPalette.ColorRole.Highlight, hlbg)
```

Then the highlight changes from black to blue.

**Comment by lennart02** [comment]
I don't know if it belongs to this issue, but if you have the Dark Mode enabled, a card with code section (<pre></pre>) also appears stil black.

**Comment by kleinerpirat** [comment]
Thanks @lennart02, yeah it's the right place :)

`QColor` doesn't accept rgba strings in CSS format, e.g. `rgba(12, 34, 56, 0.5)`. Since I introduced rgba values in the Sass maps a few PRs ago, these colors don't work for Qt elements anymore. I'm working on the fix for this.

**Comment by kleinerpirat** [comment]
Closed with #2137.
